core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerUtils.scala

```diff
@@ -17,15 +17,18 @@
 
 package org.apache.spark.scheduler.cluster.mesos
 
-import java.util.List
 import java.util.concurrent.CountDownLatch
+import java.util.{List => JList}
 
 import scala.collection.JavaConversions._
 
-import org.apache.mesos.Protos.{FrameworkInfo, Resource, Status}
-import org.apache.mesos.{MesosSchedulerDriver, Scheduler}
-import org.apache.spark.Logging
+import com.google.common.base.Splitter
+
+import org.apache.mesos.Protos._
+import org.apache.mesos.{MesosSchedulerDriver, Protos, Scheduler}
 import org.apache.spark.util.Utils
+import org.apache.spark.{Logging, SparkContext}
```

Comment:
sorry to nitpick on this again, but still not quite right.  the `o.a.spark` imports get separated.  Other `org.apache` imports are not treated specially, they are the same as `com.foo`.  Also reorder the spark imports, so it should be

```scala
import com.google.common.base.Splitter
import org.apache.mesos.Protos._
import org.apache.mesos.{MesosSchedulerDriver, Protos, Scheduler}

import org.apache.spark.{Logging, SparkContext}
import org.apache.spark.util.Utils
```